tests/fixtures/issues/issue-814.php

```diff
@@ -3499,59 +3499,6 @@
         'lite' => false,
         'standard' => true,
     ],
-    'issue-814-BQ' => [
-        'ua' => 'Mozilla/5.0 (Windows NT 10.0; WOW64; rv:41.0) Gecko/20100101 Firefox/AA5D',
```

Comment:
This is one of those useragents with the alphanumeric Firefox version (a few more follow this one). These were moved to the "invalid-versions" fixture.